extras/ffmpeg-decoder/src/ffmpeg/avcodec.rs

```diff
@@ -835,6 +844,10 @@ impl<'a> AvPacket<'a> {
 /// destination buffer.
 pub struct AvFrame(*mut ffi::AVFrame);
 
+/// SAFETY: safe because mutable pointers are exclusive en valid (implicit lifetime annotation).
+unsafe impl Send for AvFrame {}
+unsafe impl Sync for AvFrame {}
```

Comment:
Same as `AvCodecContext`, the absence of mutable aliasing should make `Sync` safe to implement.